ui/desktop/electron-app/src/helpers/app-updater.js

```diff
@@ -107,17 +107,25 @@ const displayInfoPrompt = () => {
 /**
  * Show update available prompt
  */
-const displayDownloadPrompt = (version, url) => {
+const displayDownloadPrompt = (latestVersion, url) => {
+  let detail = 'A new version is available for download.';
```

Comment:
none blocking comment: it is possible to have these strings within the translation files?

Reply:
No unfortunately this is out of the ember app, I'm not sure if we had a plan for these electron strings